Broken build due to kind

**What steps did you take and what happened:**
Travis is currently failing due to a problem with getting/installing kind.

```
49.72s$ go get sigs.k8s.io/kind
# sigs.k8s.io/kind/pkg/kustomize
../../../sigs.k8s.io/kind/pkg/kustomize/kustomize.go:108:7: undefined: k8sdeps.NewFactory
../../../sigs.k8s.io/kind/pkg/kustomize/kustomize.go:109:26: not enough arguments in call to build.NewCmdBuild
```

**What did you expect to happen:**
Successful build.

**Anything else you would like to add:**
I think they recently went to using gomodules and so we may need to tweak something to properly utilize them. (Could be wrong, but locally I hit that because I was in my GOPATH)

--- a/.travis.yml
+++ b/.travis.yml
@@ -14,7 +14,7 @@ script:
 
 before_install:
   # Download and install Kind and kubectl
-  - go get sigs.k8s.io/kind
+  - GO111MODULE=on go get sigs.k8s.io/kind
   - kind create cluster --config kind-config.yaml
   - export KUBECONFIG="$(kind get kubeconfig-path --name="kind")"
   - curl -LO https://storage.googleapis.com/kubernetes-release/release/$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)/bin/linux/amd64/kubectl

